src/lib/operators/join_hash.cpp

```diff
@@ -383,7 +388,7 @@ class JoinHash::JoinHashImpl : public AbstractReadOnlyOperatorImpl {
      */
     if (_radix_bits > 0) {
       Timer timer_clustering;
-      auto jobs = std::vector<std::shared_ptr<AbstractTask>>{};
+      std::vector<std::shared_ptr<AbstractTask>> jobs;
```

Comment:
Why did you undo the "auto to stick" style?

Reply:
Not on purpose!

Reply:
Thanks for mentioning, I was sure I already reverted it